nsv13/code/game/objects/structures/shuttlecraft_chair.dm

```diff
@@ -0,0 +1,94 @@
+/obj/item/wallframe/shuttlecraft_chair
+	name = "packaged shuttlecraft chair"
+	desc = "A box with all the parts you need to make a wall-mounted shuttlecraft chair."
+	icon = 'nsv13/icons/obj/structures/shuttlecraft_seat.dmi'
+	icon_state = "install_package"
+	materials = list(/datum/material/iron = 10000)
+	w_class = WEIGHT_CLASS_BULKY
+	result_path = /obj/structure/chair/shuttlecraft_chair
+	pixel_shift = -10
+
+/obj/structure/chair/shuttlecraft_chair
+	name = "Shuttle Seat"
+	desc = "A wall-mounted seat often found on shuttles and dropships."
+	icon = 'nsv13/icons/obj/structures/shuttlecraft_seat.dmi'
+	icon_state = "shuttle_seat"
+	anchored = TRUE
+	can_buckle = 1
+	buckle_lying = 0
+	resistance_flags = NONE
+	max_integrity = 250
+	integrity_failure = 25
+	var/opened = FALSE
+	var/mutable_appearance/armrest
+	item_chair = null
+	layer = OBJ_LAYER
+
+/obj/structure/chair/shuttlecraft_chair/Initialize(mapload, ndir, building)
+	. = ..()
+	if(building)
+		setDir(ndir)
+
+/obj/structure/chair/shuttlecraft_chair/Initialize()
+	armrest = GetArmrest()
+	armrest.layer = ABOVE_MOB_LAYER
+	return ..()
+
+/obj/structure/chair/shuttlecraft_chair/proc/GetArmrest()
+	return mutable_appearance('nsv13/icons/obj/structures/shuttlecraft_seat.dmi', "belt")
+
+/obj/structure/chair/shuttlecraft_chair/Destroy()
+	QDEL_NULL(armrest)
+	return ..()
+
+/obj/structure/chair/shuttlecraft_chair/post_buckle_mob(mob/living/M)
+	. = ..()
+	update_armrest()
+
+/obj/structure/chair/shuttlecraft_chair/proc/update_armrest()
+	if(has_buckled_mobs())
+		add_overlay(armrest)
+		AltClick(null)
```

Comment:
this doesn't need to be here, it'll always return on lines 67 and 68 because null isn't a mob/living. 
just adding the has_buckled_mobs() to /obj/structure/chair/shuttlecraft_chair/AltClick(mob/living/user) was enough.